src/gcodeExport.cpp

```diff
@@ -816,7 +816,8 @@ void GCodeExport::writeZhopStart(int hop_height)
     if (hop_height > 0)
     {
         isZHopped = hop_height;
-        *output_stream << "G1 F" << PrecisionedDouble{1, current_max_z_feedrate * 60} << " Z" << MMtoStream{current_layer_z + isZHopped} << new_line;
+        currentSpeed = current_max_z_feedrate * 60;
```

Comment:
Could you tell me how this is any different from what it already did? As far as I see this is just a different way of saying the exact same thing (eg; instead of doing current_max_z_feedrate * 60 inline, you first assign it to a variable)